app/src/components/CollectionView.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Alignment, Box, Direction, Image, Stack, Text } from '@kibalabs/ui-react';
+
+import { Collection } from '../client/resources';
+
+export interface CollectionViewProps {
+  collection: Collection;
+}
+const defaultImage = '/assets/icon.png';
+
+export const CollectionView = (props: CollectionViewProps): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} shouldAddGutters={true}>
+      <Text>Part of</Text>
```

Comment:
this should not be part of this view. just like "owned by" is not part of the account view

Reply:
ping on this